cpp/models/abm/world.cpp

```diff
@@ -110,7 +110,8 @@ void World::migration(TimePoint t, TimeSpan dt)
             auto& trip            = m_trip_list.get_next_trip();
             auto& person          = m_persons[trip.person_id];
             auto current_location = person->get_location();
-            if (!person->is_in_quarantine() && current_location == get_individualized_location(trip.migration_origin)) {
+            if (!person->is_in_quarantine() && person->get_infection_state(t) != InfectionState::Dead &&
```

Comment:
Sorry, this is only for the trips. That is fine then, unless one plans to integrate trips to the Cemetery as a visitor, but in that case one would have to consider multiple cemeteries certainly and this is not what we aim to do here.
Since we solved quarantine the same way with migration rules, this is consistent too. It is only questionable to assign the cemetery to all people since we consider the cemetery to be a unique hub for all dead people to "meet".

Reply:
Yes, the code prevents a dead Person moving out of Cemetery in case its has a trip after dead. I guess I will leave it like that.